seleniumwire/webdriver.py

```diff
@@ -14,6 +14,12 @@
 from seleniumwire import backend
 from seleniumwire.inspect import InspectRequestsMixin
 
+try:
+    # noinspection PyUnresolvedReferences
+    from undetected_chromedriver import ChromeOptions
```

Comment:
This line generates a flake8 error due to the duplicated import. Would you be able to add `# noqa`:

```
from undetected_chromedriver import ChromeOptions  # noqa
```